documentation/Releasing.md

```diff
@@ -9,32 +9,15 @@ names with the appropriate cases.
 - [ ] If using profile-guided optimization (PGO), make sure that the `PlayerTest.Benchmark` test contains the latest journal file name and has been merged as the last pull request.
 - [ ] Check that the release name, release date, and lunation number for the following update, used by the update reminder, are correctly set in the adapter.
 - [ ] Make sure that the project can be built and tested under Linux.
-- [ ] Create and push a tag for the current state of mockingbirdnest/master, where `"yyyyMMddHH"` is the date of the new moon in UT1:
+- [ ] Run the following command, where:
+  - `yyyyMMddHH` is the date of the new moon in UT1,
+  - `"<language>"` is the language code for the language in which `<Mathematician>` is written,
```

Comment:
Why the double quoting around `language`?

Reply:
What is the case of `Mathematician`?  It should be documented.

Reply:
It is (still) documented above.